std/experimental/allocator/building_blocks/stats_collector.d

```diff
@@ -729,7 +729,7 @@ public:
         private void addPerCall(string f, uint n, names...)(ulong[] values...)
         {
             import std.array : join;
-            enum uint mask = mixin("Options."~[names].join("|Options."));
+            enum ulong mask = mixin("Options."~[names].join("|Options."));
```

Comment:
Assigning ulong to uint.

Reply:
This is known at compile time, isn't it ? So it should not be a problem.

Reply:
The idea is that if the user has forced a type upon an enum, it should always be taken into account.

Reply:
That sounds like a very arbitrary limitation without apparent benefit.